<launch>

    <node pkg="gvins_dataset_toolkit" type="bag2rinex" name="bag2rinex" output="screen">
        <!-- 设置节点私有参数 -->
        <param name="input_bag_path" type="string" value="/home/wxp/bigbag.bag"/>
        <param name="output_obs_path" type="string" value="/home/wxp/convert_ws/src/GVINS-Dataset/data/ephemeris_rinex/bigbag.obs"/>
        <param name="output_nav_path" type="string" value="/home/wxp/convert_ws/src/GVINS-Dataset/data/ephemeris_rinex/bigbag.nav"/>
        <param name="obs_flag" type="bool" value="true"/>
        <param name="nav_flag" type="bool" value="true"/>
    </node>
    <!-- <include file="$(find another_package)/launch/another.launch" /> -->
</launch>